src/main/java/org/galatea/starter/domain/IexHistoricalPrices.java

```diff
@@ -16,4 +15,41 @@
   private BigDecimal open;
   private long volume;
   private String date;
+
+  protected IexHistoricalPrices() {}
```

Comment:
Why protected?

Reply:
You're right in the sense that it isn't necessary to have here, but it is necessary for the database object (needed for JPA according to Spring guide). I'll get rid of it here (left over from when I was trying to force this into the database instead of creating a new object type).